index.bs

```diff
@@ -3997,6 +3994,7 @@ partial interface MLGraphBuilder {
         <div class="issue">
             The shape of |input|, |weight| or |recurrentWeight| could be also checked here.
         </div>
+    1. Require the shape of each tensor to match its expected rank, or else [=exception/throw=] a "{{DataError}}" {{DOMException}}: the rank of |input|, |weight|, |recurrentWeight|, |options|.{{MLLstmOptions/initialHiddenState}}, and |options|.{{MLLstmOptions/initialCellState}} is 3; and the rank of |options|.{{MLLstmOptions/bias}}, |options|.{{MLLstmOptions/recurrentBias}}, and |options|.{{MLLstmOptions/peepholeWeight}} is 2.
```

Comment:
Some problems with this:
- These checks aren't guarded by "exists" check for the options members.
- The options members are checked below, which would make parts of the overall algorithm redundant.
- Prefer imperative style, i.e. "if... then..."

So I'd suggest copying what [gru()](https://webmachinelearning.github.io/webnn/#dom-mlgraphbuilder-gru)  has:

```
1. If the [=rank=] of |input| or |weight| or |recurrentWeight| is not 3, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
```

Reply:
The GRU wording is indeed sufficient an consistent.